### ISSUE TITLE
Database creation issues

### ISSUE THREAD

**Issue by jhnath21** [DerrickWood/kraken2#892]
With NCBI changing the FTP site, many of the database building fail or are not being updated like expected.  For instance, to build the NT database, it is pulling the nt.gz file and uncompressing it.  However, per the FTP site README.txt file, this file is no longer being updated on the FTP site and that NCBI recommends using the blastdbcmd process to extract out the sequences into fasta format.

**Comment by ch4rr0** [comment]
You are correct, our current set up downloads a no-longer-updated nt/nr.fa file. You can build an updated FASTA of nt using the following commands:

``` shell
update_blastdb --source ncbi "nt" --num_threads 10 --decompress
blastdbcmd -entry all -db "nt" -out "nt.fna"
```

We are considering updating our current code to use these external tools. It will however require that the user has the necessary executables installed on the host machine. We do publish a quarterly `core_nt` database available here: https://genome-idx.s3.amazonaws.com/kraken/k2_core_nt_20240904.tar.gz
